openmetadata-ui/src/main/resources/ui/cypress/constants/constants.js

```diff
@@ -58,26 +58,30 @@ export const SEARCH_ENTITY_TABLE = {
     displayName: 'raw_customer',
     entity: MYDATA_SUMMARY_OPTIONS.tables,
     serviceName: 'sample_data',
+    entityType: 'Table',
   },
   table_2: {
     term: 'fact_session',
     displayName: 'fact_session',
     entity: MYDATA_SUMMARY_OPTIONS.tables,
     serviceName: 'sample_data',
     schemaName: 'shopify',
+    entityType: 'Table',
   },
   table_3: {
     term: 'raw_product_catalog',
     displayName: 'raw_product_catalog',
     entity: MYDATA_SUMMARY_OPTIONS.tables,
     serviceName: 'sample_data',
     schemaName: 'shopify',
+    entityType: 'Table',
   },
   table_4: {
     term: 'dim_address',
     displayName: 'dim_address',
     entity: MYDATA_SUMMARY_OPTIONS.tables,
     serviceName: 'sample_data',
+    entityType: 'Table',
```

Comment:
why do we need entityType? All the constants are already segregated right?

Reply:
I'm using that entityType to select the entity type while doing global search,so many tests were failing with the  upgrade tests due to not getting the results for a particular entity. e.g. after typing `orders` to fine that named topic, other suggestions were coming except for topic entities.